src/Backend/Teradata/Helper/DropTableTrait.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Db\ImportExport\Backend\Teradata\Helper;
+
+use Keboola\Db\ImportExport\Backend\Teradata\ToFinalTable\SqlBuilder;
+
+trait DropTableTrait
```

Comment:
tady by se hodil jiny nazev, treba `CommandsTrait` nebo `DbCommandsTrait`...

Reply:
ty jo nevim, nemam vyhraneny nazor, muj plan s touto traitou je aby delala jen toto. nic vic, nic min, tak bych ji i tak nechal pojmenovanou. Nevidim v ostatnich traitach nejakou konvenci... 
![image](https://user-images.githubusercontent.com/13363655/202022069-310e3fd2-4097-435d-808d-7de424068f33.png)

Reply:
OK, az to bude potreba, tak to muzem treba prejmenovat...